src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { loadData } from './redux/reducer';
+import CountryDetails from './components/CountryDetails';
+import Main from './components/Main';
+import NavBar from './components/NavBar';
+
+const App = () => {
+  const dispatch = useDispatch();
+  const selectedCountry = useSelector((state) => state.country.country);
+
+  useEffect(() => {
+    dispatch(loadData());
+  }, []);
+
+  return (
+    <Router>
+      <div className="box-border text-white bg-lightpink pb-3 mt-0">
+        <NavBar />
+        <Routes>
+          <Route exact path="/" element={<Main />} />
+          <Route
+            exact
+            path="/Details"
+            element={<CountryDetails selectedCountry={selectedCountry || ''} />}
+          />
+        </Routes>
+      </div>
+    </Router>
```

Comment:
As requested by the previous reviewer, let's add the path to the other 2 pages in the project here.
- The category item detail page has its unique route within the SPA. Let's make sure when we navigate(click) to a specific continent, our URL adds its unique path, and when to navigate(click) to a specific country our URL adds its unique path.